src/StratisMasternodeDashboard/wwwroot/js/nodeevent.js

```diff
@@ -24,6 +24,10 @@ connection.on("receiveEvent", function (message) {
         }
     }
 
+    if (message.nodeEventType.includes("Stratis.Bitcoin.Features.MemoryPool.TransactionAddedToMemoryPoolEvent")) {
+        alert("Mempool")
```

Comment:
Can you now set the mempool size from the event?

Reply:
OK let me know when your node is fully synced, then we can test this by submitting a transaction onto the network.